lldb-eval/parser_context.h

```diff
@@ -11,6 +11,7 @@ enum class ErrorCode : unsigned char {
   kOk = 0,
   kInvalidExpressionSyntax,
   kInvalidNumericLiteral,
+  kInvalidCharLiteral,
```

Comment:
Unfortunately adding new error codes is a breaking API change and it will break the Visual Studio extension :(
Let's use `kInvalidNumericLiteral` for now and leave a TODO, please, saying that it should be a separate error code.